backend/src/main/java/ai/verta/modeldb/datasetVersion/DatasetVersionServiceImpl.java

```diff
@@ -57,20 +60,28 @@ public DatasetVersionServiceImpl(ServiceSet serviceSet, DAOSet daoSet) {
   }
 
   private void saveAuditLogs(
-      UserInfo userInfo, String action, List<String> resourceIds, String metadataBlob) {
+      Optional<UserInfo> userInfo,
+      ModelDBServiceActions action,
+      List<String> resourceIds,
+      String request,
+      String response,
+      Long workspaceId) {
```

Comment:
I see this is `null` everywhere. It actually must be provided or the search on the UAC side will be broken.

Reply:
PR https://github.com/VertaAI/modeldb/pull/1981